src/app/workflow/launch-third-party/multi-cloud-launch/multi-cloud-launch.component.spec.ts

```diff
@@ -29,4 +29,11 @@ describe('MultiCloudLaunchComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('Should construct url with https protocol', () => {
+    expect(component.constructUrlWithHttpsProtocol('usegalaxy.ca')).toBe('https://usegalaxy.ca');
+    expect(component.constructUrlWithHttpsProtocol('subdomain.usegalaxy.ca')).toBe('https://subdomain.usegalaxy.ca');
+    expect(() => component.constructUrlWithHttpsProtocol('http://foobar.com')).toThrow();
```

Comment:
Amused that this triggers a security alert (alert can be ignored of course for other reviewers).
May be possible to put in a comment to have sonarcloud ignore this line